examples/conditions/pull_approve.js

```diff
@@ -333,17 +333,32 @@ function reviewed({
       if (satisfied) {
         file.designatedReviewers.push(
           ..._.map(team, (unused, key) => ({builtin: 'fulfilled', scope})));
-        if (unrequest) unrequestedTeams.push(..._.keys(team));
-      } else {
-        requiredTeams.push(..._.keys(team));
-        if (request && !_.some(file.revisions, 'reviewed')) requestedTeams.push(..._.keys(team));
       }
     } else {
       file.designatedReviewers.push({builtin: 'anyone', omitBaseChanges});
       if (satisfied) file.designatedReviewers.push({builtin: 'fulfilled'});
     }
   }
 
+  if (team) {
+    if (allSatisfied) {
+      if (unrequest) unrequestedTeams.push(..._.keys(team));
+    } else {
+      requiredTeams.push(..._.keys(team));
+      const engagedReviewersScore = _(pr.reviewers)
```

Comment:
Should filter out reviewer that are not `participating` otherwise I'm assuming lurkers in the team would cause to unrequest too fast